app/store/reducers/appState/data.js

```diff
@@ -1,7 +1,7 @@
 import { SET_CURRENT_USER_ID } from "../../constants/actionTypes";
 
 const initialState = {
-  currentUserID: 0
+  currentUserID: undefined
```

Comment:
i am not sure undefined is right 
i think in js we should avoid to use undefined